### DISCUSSION TITLE
OKD update not showing latest release

### DISCUSSION THREAD

**Discussion by bobby0724** [okd-project/okd#751] [Q&A]
**Describe the bug**
OKD UPI installation not showing latest release to peform upgrade

**Version**
4.7.0-0.okd-2021-06-04-191031

**How reproducible**
From version 4.7.0-0.okd-2021-06-04-191031 update is not showing 4.7.0-0.okd-2021-07-03-190901

![image](https://user-images.githubusercontent.com/30838789/124659547-b33dbf00-de6a-11eb-91a8-26e7f72cc753.png)

**Log bundle**

**Comment by kai-uwe-rommel** [comment]
To me it looks like the update graph is currently configured so that you can skip only release.
Or perhaps this is a barrier release that needs to be passed over.
I had a cluster with 2021-06-04 as well. After upgrading it to 2021-06-19 it was then possible to upgrade to 2021-07-03.

**Comment by vrutkovs** [comment]
See `Upgrades to` section on [release page](https://amd64.origin.releases.ci.openshift.org/releasestream/4-stable/release/4.7.0-0.okd-2021-06-04-191031), it lists all available upgrades.

In 4.7.0-0.okd-2021-06-13-090745 we changed the way `machine-os-content` is being built, moving from FCOS commit + unpacked RPMs to a proper coreos-assembler built image. This is a significant internal change and in order to avoid regressions users must upgrade to 06-13 first to proceed.

**Reply by bobby0724** [reply]
thanks
